app/js/ffxivcraftmodel.js

```diff
@@ -279,8 +276,8 @@ function ApplyModifiers(s, action, condition) {
         progressIncreaseMultiplier += calcNameOfElementsBonus(s);
     }
 
-    if (AllActions.innovation.shortName in s.effects.countDowns) {
```

Comment:
Has the effects of Innovation changed?

Reply:
Yes:
Effect changed from **"Increases efficiency of Synthesis and Touch actions by 20% for the next four steps"** to **"Increases efficiency of Touch actions by 50% for the next four steps."**